tests/components/crownstone/test_config_flow.py

```diff
@@ -29,21 +28,21 @@
 from tests.common import MockConfigEntry
 
 
-@pytest.fixture(name="crownstone_setup", autouse=True)
-def crownstone_setup():
-    """Mock Crownstone entry setup."""
-    with patch(
-        "homeassistant.components.crownstone.async_setup_entry", return_value=True
```

Comment:
We still want to patch the integration setup function on all create entry results in the config flow tests (not options flow tests) to avoid setting up the whole integration.